.aspect/workflows/bazelrc

```diff
@@ -1,11 +1,3 @@
 # build without the bytes
 common --remote_download_outputs=minimal
 common --nobuild_runfile_links
-
-# remote execution
-common --extra_execution_platforms=@aspect_rules_terser//.aspect/workflows:x86_64_linux_remote
```

Comment:
Should also remove the platform definition BUIDL file in the `.aspect/workflows` folder